pkgs/development/python-modules/pomegranate/default.nix

```diff
@@ -13,39 +13,63 @@
   scipy,
   pyyaml,
   cython,
+  scikit-learn,
+  torch,
+  apricot-select,
+  pytestCheckHook,
 }:
-
 buildPythonPackage rec {
   pname = "pomegranate";
-  version = "0.14.8";
+  version = "1.1.0";
   pyproject = true;
 
   src = fetchFromGitHub {
-    repo = pname;
     owner = "jmschrei";
-    # no tags for recent versions: https://github.com/jmschrei/pomegranate/issues/974
-    rev = "refs/tags/v${version}";
-    hash = "sha256-PoDAtNm/snq4isotkoCTVYUuwr9AKKwiXIojUFMH/YE=";
+    repo = "pomegranate";
+    tag = "v${version}";
+    hash = "sha256-p2Gn0FXnsAHvRUeAqx4M1KH0+XvDl3fmUZZ7MiMvPSs=";
   };
 
-  nativeBuildInputs = [ setuptools ];
+  build-system = [ setuptools ];
 
-  propagatedBuildInputs = [
+  dependencies = [
     numpy
     joblib
     networkx
     scipy
     pyyaml
     cython
+    scikit-learn
+    torch
+    apricot-select
+  ];
+
+  pythonImportsCheck = [
+    "pomegranate"
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pytestFlagsArray = [
+    "tests/"
   ];
 
-  # https://github.com/etal/cnvkit/issues/815
-  passthru.skipBulkUpdate = true;
+  disabledTestPaths = [
+    "tests/distributions/test_categorical.py"
```

Comment:
Add a comment with the reason for disabling them, e.g. the error message they fail with.